tests/sizeof_tcb/Makefile

```diff
@@ -1,4 +1,7 @@
 APPLICATION = sizeof_tcb
 include ../Makefile.tests_common
 
+# optionally disable messaging
+#DISABLE_MODULE += msg
```

Comment:
Ah. Nope, IMHO sizeof_tcb should show the tcb size in the general case. disabling messaging seems more on the special side.